src/adafruit_blinka/__init__.py

```diff
@@ -74,6 +81,41 @@ def unlock(self):
             self._locked = False
 
 
+def load_settings_toml(*, return_toml=False):
```

Comment:
@justmobilize does the wildcard `*` arg here do something specific?

Reply:
Ahh, tinkering in REPL a little I see it basically forces you to pass the argument by name only.

Reply:
Yup. I find it helpful for methods like this. Makes updating the function signature easier if ever needed